firstvoices/backend/serializers/character_serializers.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
+
+from backend.models.characters import Character, CharacterVariant, IgnoredCharacter
+from backend.models.dictionary import DictionaryEntry
+from backend.serializers.dictionary_serializers import DictionaryEntryDetailSerializer
+from backend.serializers.fields import SiteHyperlinkedIdentityField
+
+
+class IgnoredCharacterSerializer(serializers.ModelSerializer):
+    url = SiteHyperlinkedIdentityField(view_name="api:ignored_characters-detail")
+    site = serializers.StringRelatedField()
+
+    class Meta:
+        model = IgnoredCharacter
+        fields = ["url", "id", "title", "site"]
+
+
+class CharacterVariantSerializer(serializers.ModelSerializer):
+    base_character = serializers.StringRelatedField()
+
+    class Meta:
+        model = CharacterVariant
+        fields = ["id", "title", "base_character"]
```

Comment:
Minor: since this serializer is used within a CharacterDetailSerializer, I don't think it needs to include the base_character

Reply:
I'm curious whether we need the id, also. An object containing a "title" is probably enough